pylib/mypy.ini

```diff
@@ -3,7 +3,12 @@ python_version = 3.7
 pretty = false
 no_strict_optional = true
 show_error_codes = true
+
```

Comment:
Why the space?

Reply:
I thought it'd be easier to group the settings into "display settings" and "typecheck setings" (although I see now that I missed `no_strict_optional`). Is this something you'd want, or would you like me to revert the new line here?

Reply:
python_version is not a display setting either - probably simplest to remove the gap.